src/net/java/sip/communicator/service/protocol/StunServerDescriptor.java

```diff
@@ -248,8 +250,16 @@ public void storeDescriptor(Map<String, String> props, String namePrefix)
             props.put(namePrefix + STUN_PORT, Integer.toString( getPort() ));
 
         if (getUsername() != null && getUsername().length > 0)
-            props.put(namePrefix + STUN_USERNAME,
-                      StringUtils.getUTF8String(getUsername()));
+            try
+            {
+                props.put(namePrefix + STUN_USERNAME,
```

Comment:
toEncodedString

Reply:
It could not to resolve symbol for some reason.
Here is the output:
    [javac] D:\Source\Repos\jitsi\src\net\java\sip\communicator\plugin\jabberaccregwizz\IceConfigPanel.java:213: error: cannot find symbol
    [javac]                         userName = StringUtils.toEncodedString(stunServer.getUsername(), StandardCharsets.UTF_8);
    [javac]                                               ^
    [javac]   symbol:   method toEncodedString(byte[],Charset)

Reply:
Please get rid of the try/catch